react/features/participants-pane/functions.js

```diff
@@ -235,6 +236,11 @@ export function getSortedParticipantIds(stateful: Object | Function): Array<stri
         dominant.push(dominantId);
     }
 
+    if (conference.isHidden()) {
```

Comment:
Do we really need to do this? We could disable the ability to raise the hand instead perhaps?

Reply:
this is in order to hide self in participants pane. Not related to raise hand